patch.js

```diff
@@ -0,0 +1,13 @@
+const fs = require('fs');
```

Comment:
This can be tricky, you're trusting on path consistency and changing things in distribution files. Isn't there really other way to change this specific config

Reply:
When I was looking into the issue https://github.com/angular/angular-cli/issues/1548, I chose this as the most viable of the proposed solutions, some others required some extra third party dependencies that I would rather avoid using.

Since then I didn't really look into it since it was working. I will have to go through the thread once again and see if there is a better solution or if this has been fixed.

Reply:
So there is no upstream fix for these it seems, I started getting the error as soon as I removed the `patch.js` from running.

I won't disagree that it is tricky, however this webpack configuration is hidden inside angular's and at least from what I understood, you are not supposed to customize it, so there is no way to change it other than patching it.

The reason that I chose this approach is that it was the only solution that didn't require installing extra thirdparty dependencies.

Reply:
Worst case you could eject, but I think it's not worth the trouble right now, seems like this is affecting enough people to be fixed quite soon, so as a temporary workaround could stay, as soon as you include specific notes and TODOs linking the issue in question. Thank you

Reply:
I added the comment to the patch. Is that ok ?